frame/evm-system/src/lib.rs

```diff
@@ -248,8 +222,8 @@ impl<T: Config> fp_evm::AccountProvider for Pallet<T> {
 		let _ = Self::create_account(who);
```

Comment:
Should this one be called `on_account_created`?

Reply:
It checks whether the account already exists or not. If not, it inserted the account and calls on_created_account